src/main/kotlin/no/nav/klage/dokument/clients/kabalsmarteditorapi/KabalSmartEditorApiClient.kt

```diff
@@ -152,14 +154,20 @@ class KabalSmartEditorApiClient(
 
     fun deleteCommentWithPossibleThread(
         documentId: UUID,
-        commentId: UUID
+        commentId: UUID,
+        behandlingTildeltIdent: String?
     ) {
-        kabalSmartEditorApiWebClient.delete()
+        kabalSmartEditorApiWebClient.method(HttpMethod.DELETE)
             .uri { it.path("/documents/$documentId/comments/$commentId").build() }
             .header(
                 HttpHeaders.AUTHORIZATION,
                 "Bearer ${tokenUtil.getSaksbehandlerAccessTokenWithKabalSmartEditorApiScope()}"
             )
+            .bodyValue(
```

Comment:
Denne er jo litt vrien muligens. Delete er vel ikke kjent for å ville ha en `body`. Vet ikke om det blir penere med en `post` da? Kan jo teoretisk bli noen problemer gjennom brannmurer etc.?

Reply:
Enig, endret det til `POST` nå.